_includes/docs/v2.25/setup-script-credentials-gke.md

```diff
@@ -2,6 +2,10 @@
 . ./helpers/setup.sh -p PROJECT_ID -o ORG_ID -k
 ```
 
-This will create a service account called `cloud-foundation-forseti-<suffix>`,
-assign it the proper roles, and download the service account credentials to
-`${PWD}/credentials.json`.
+This will create a service account called `cloud-foundation-forseti-<suffix>`
+and assign it the necessary roles. 
+
+Set and export the `GOOGLE_APPLICATION_CREDENTIALS` variable to the newly
+created credential.json as shown below.
```

Comment:
credential.json --> credentials.json

Also same comment as above

Reply:
Removed this sentence as this is covered in setup.sh script.